viewer/package.json

```diff
@@ -2,8 +2,8 @@
   "name": "@cognite/reveal",
   "version": "0.1.0",
   "description": "Reveal streaming component",
-  "main": "index.js",
-  "types": "index.d.ts",
+  "main": "dist/index.js",
```

Comment:
Why is this being changed? The package.json is moved into the dist folder when built, so this location is wrong. 

If I recall correctly, the reason the package.json is moved to dist is because TypeScript would pick up other .ts files in src when we linked the examples to viewer. By linking to viewer/dist, we no longer picked up spurious files.

Might be that we do not need to move it after we changed the ts-loader options to

```
            options: {
              onlyCompileBundledFiles: true,
            },
```
but that seems unrelated to this change. Especially since examples still point to viewer/dist.

Reply:
Hmmm, I did this because I had to import reveal using "@cognite/reveal/dist". Not sure how or why this happened, but changing this didn't seem to break anything?

Reply:
That is weird. I think it might break things when running npm pack, though. I will test on my machine.

Reply:
Ah, we removed the `cp package.json dist` line from the build. That might be why you ran into problems. However, it seems like it works now either way, so we can keep this change.